src/fmu/dataio/models/global_configuration.py

```diff
@@ -0,0 +1,27 @@
+"""
+Provides classes for managing and validating global configuration
+settings in an application. These classes ensure essential settings
+are defined and maintained consistently.
+"""
+
+from __future__ import annotations
+
+from typing import Dict, List, Optional
+
+from pydantic import BaseModel
+
+
+class MinimumGlobalConfigurationStratigraphy(BaseModel):
+    name: str
+    stratigraphic: bool
+    alias: Optional[List[str]] = None
+    stratigraphic_alias: Optional[List[str]] = None
+
+
+class MinimumGlobalConfiguration(BaseModel):
+    """A minimum check/validation of the static global_config."""
+
+    access: Dict
+    masterdata: Dict
```

Comment:
Can this use the masterdata schema from the meta-module? Yes?

Reply:
Example of expected masterdata block from config: https://github.com/equinor/fmu-drogon/blob/19e3942ee91d915ece073f22e87c0a3f292cabc4/fmuconfig/input/_masterdata.yml